### PR TITLE
Add dataio export

### PR DESCRIPTION
**Pull request by jcrivenaes**
Support for fmu-dataio #52:
* Try to do this as lean as possible, i.e. a minimal change in config for the user
* Only map output, no plots are supported for dataio/SUMO (the idea is that Webviz/REP/etc do the plotting)
* fmu-dataio is still in beta, so implementation here must adapt to newer version of fmu-datio

The commit is a bit messy as the code in this is repo is "a candidate for a heavy refactoring". That refactoring should come at some later stage, but some fixes were required.

### CODE REVIEW

File: scripts/setup_functions.py [hunk 1]
```diff
@@ -91,6 +93,12 @@ def run(self):
                 print(f"Remove file {pfil}")
                 os.unlink(pfil)
 
+        for fil_ in CleanUp.CLEANSPECIAL:
+            print(fil_)
```
Comment (berland) [inline scripts/setup_functions.py:97]: debugging leftover?

File: src/xtgeoapp_grd3dmaps/avghc/_compute_hcpfz.py [hunk 2]
```diff
@@ -110,17 +105,16 @@ def _get_hcpfz_ecl(config, initd, restartd, dates, hcmode, filterarray):
 
     for cdate in cdates:
         cdate = str(cdate)
-        logger.debug("cdate is: %s", cdate)
+        logger.info("cdate is: %s", cdate)
```
Comment (berland) [inline src/xtgeoapp_grd3dmaps/avghc/_compute_hcpfz.py:108]: Is "`cdate`" a concept the user understands (now that it is upgraded from debug to info)?

Comment (jcrivenaes) [reply src/xtgeoapp_grd3dmaps/avghc/_compute_hcpfz.py:108]: Will fix

**Review by berland — APPROVED** [review]
lgtm

**Pull request merged**